### ISSUE TITLE
jq command stopped working on azure-cli:latest Docker image (2.64.0)

### ISSUE THREAD

**Issue by sruthip-tomtom** [Azure/azure-cli#29830]
### Describe the bug

We are using jq to parse JSON data. That worked fine up to Azure CLI 2.63.0 Docker image, but since today with 2.64.0 it fails. We are using azure-cli:latest

However on specifying the previous version 2.63.0 as the azcliversion the command ran successfully

### Related command

`az databricks workspace show --subscription $SUBSCRIPTION_ID  --resource-group $RESOURCE_GROUP_NAME  --name $NAME | jq -r  '.workspaceUrl'`

### Errors

```
/home/runner/_work/_temp/AZ_CLI_GITHUB_ACTION_1725347898679.sh: line 9: jq: command not found
WARNING: Preview version of extension is enabled by default for extension installation now. Will be disabled in future release. 
WARNING: Please run 'az config set extension.dynamic_install_allow_preview=true or false' to config it specifically. 
WARNING: The command requires the extension databricks. It will be installed first.
Exception ignored in: <_io.TextIOWrapper name='<stdout>' mode='w' encoding='utf-8'>
BrokenPipeError: [Errno 32] Broken pipe
Error: Error: az cli script failed.
```

### Issue script & Debug output

the jq package is not found in the docker container image 2.64.0

### Expected behavior

The workflow should have ran successfully 

### Environment Summary

azure-cli                         2.64.0

**Comment by foxt** [comment]
Seems like this also broke the `azure/acr-build` GitHub Action

https://github.com/Azure/acr-build/issues/19

**Comment by superherointj** [comment]
The `jq` removal broke my pipeline. The suggested solution of using `azure-cli`'s [--query](https://learn.microsoft.com/en-us/cli/azure/use-azure-cli-successfully-query?tabs=concepts%2Cbash) parameter is a bad solution. `yq` is more flexible and decoupled from `azure-cli`. I think, this is a mistake.

**Comment by bebound** [comment]
`jq` will be added in Azure Linux image in next release.

**Comment by subha-sa** [comment]
@bebound is this fixed in 2.64.0 version?

**Comment by superherointj** [comment]
> @bebound is this fixed in 2.64.0 version?

No. Waiting for next release.
